src/app/Translator.js

```diff
@@ -3,18 +3,36 @@ import {connect} from 'react-redux'
 import {IntlProvider, addLocaleData} from 'react-intl';
 import en from 'react-intl/locale-data/en';
 import es from 'react-intl/locale-data/es';
+import ru from 'react-intl/locale-data/ru';
+import fr from 'react-intl/locale-data/fr';
+import it from 'react-intl/locale-data/it';
 import {DEFAULT_LANGUAGE} from 'app/client_config';
 import tt from 'counterpart';
 
-addLocaleData([...en, ...es]);
+addLocaleData([...en, ...es, ...ru, ...fr, ...it]);
```

Comment:
man I love splats

if you wanted to implement splats in algol you'd use the token YADDAYADDA

Reply:
You are making me to pronounce ... as YADDAYADDA for the entire life :)